controllers/gatekeeper_controller.go

```diff
@@ -58,9 +63,9 @@ var (
 		"apiextensions.k8s.io_v1beta1_customresourcedefinition_constraintpodstatuses.status.gatekeeper.sh.yaml",
 		"~g_v1_secret_gatekeeper-webhook-server-cert.yaml",
 		"~g_v1_serviceaccount_gatekeeper-admin.yaml",
+		roleFile,
```

Comment:
Did it become necessary to reorder this file?

Reply:
yeah, the e2e test was failing until I reordered it

Reply:
Do you know what exactly was causing the failure?

Reply:
No, but I'd guess trying to create the role binding before the role was causing issues

Reply:
Prior to this change, the role was being created before the role binding that comes right after it, so I don't think that should have caused it. Would you be able to capture the error and file an issue if necessary?